mlkem/poly.h

```diff
@@ -38,17 +38,6 @@ typedef struct
   int16_t coeffs[MLKEM_N >> 1];
 } poly_mulcache;
 
-#define scalar_compress_d1 MLKEM_NAMESPACE(scalar_compress_d1)
```

Comment:
There are probably more examples for this? Any particular reason why this change was made?

Reply:
I just removed it here because I saw it. 
I've dropped it here and added it to my todo list to remove it later for all static functions.